### ISSUE TITLE
Entry Display Garbled for One User of Same Database

### ISSUE THREAD

**Issue by HUGHUSR** [keepassxreboot/keepassxc#2801]
[TIP]:  # ( Provide a general summary of the issue in the title above ^^ )
[TIP]:  # ( DO NOT include screenshots of your actual database! )

Configuration: I have multiple users on my Mac. I login to only one of them at a time. I have several .kdbx files that are on a flash drive, and the databases are accessible my all users.

Using KeePassXC 2.3.2, and logged in as "User A", I had a kdbx database open when a app (that has no access to or use of my password databases) crashed, and took my MBP out with it. After a reboot, ALL databases accessed by "User A" now showed the "Notes" at the bottom of each entry, despite having "Hide Notes" checked in Preferences. If I hid an individual entry's note, as soon as I accessed that entry, the note reappeared. Also, the only DB columns I display are Title, Username, URL, and Notes.

Logging in as ANY different user, the app behaves correctly.

I then decided to upgrade to KeePassXC 2.3.4. I ran AppZapper before the install, to make sure I had a clean install. I also ran a $(find ~ -iname '*keepassxc*') to search for files that AppZapper missed and found none. I then did the new version install.

Same behavior for Notes, but again, only for "User A" and no other user. NEW wrong behavior, is that the column display correctly has "Hide Passwords" checked, but started-out passwords still appear. Toggle of the setting changes nothing.

Since this only impacts a single user, I've clearly missed wiping something somewhere! What and where?

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )

## Current Behavior
[NOTE]: # ( Tell us what actually happens )

## Possible Solution
[NOTE]: # ( Not required, but suggest a fix/reason for the bug )

## Steps to Reproduce
[NOTE]: # ( Provide a link to a live example, or an unambiguous set of steps to )
[NOTE]: # ( reproduce this bug. Include code to reproduce, if relevant )
1.
2.
3.

## Context
[NOTE]: # ( How has this issue affected you? What unique circumstances do you have? )

## Debug Info
[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - VERSION
Revision: REVISION

Libraries:
- LIBS

Operating system: OS
CPU architecture: ARCH
Kernel: KERNEL

Enabled extensions:
- EXTENSIONS

**Comment by HUGHUSR** [comment]
Uh... on my "find" command -- I had an asterisk on either side of the name I was searching on. Posting the entry killed those to asterisks. Please mentally correct the find command when you read the original posting.

**Comment by droidmonkey** [comment]
You clearly have a settings storage issue. You can clear that on macos by deleting the file located at ```/users/[username]/Library/Application Support/keepassxc/keepassxc.ini```

**Comment by HUGHUSR** [comment]
Hi,
I tried quitting the app, killing the file (rm ~/Library/Application
Support/keepassxc/keepassxc.ini) then restarted the app. It did NOT
fix the problem.
So, I tried an experiment: I took the ini file from a user that did
not have the problem and used it replace the user with a problem's
file. That DID fix the problem
I have zero clue why the first fix didn't work but the second one did.
But, at least my problem is now fixed.
Thanks for your help!
HU
On 3/18/2019 at 12:16 PM, "Jonathan White"  wrote:

	You clearly have a settings storage issue. You can clear that on
macos by deleting the file located at
/users/[username]/Library/Application Support/keepassxc/keepassxc.ini
